StatCounter.py

```diff
@@ -353,7 +351,7 @@
 		nbins = len(histogram)
 
 		for start in range(len(stallCounter)):
-			if stallCounter[start] >= histogram[0][0]-binSize/2:
+			if stallCounter[start][0] >= histogram[0][0]-binSize/2:
```

Comment:
Explain?

Reply:
From Python 2:
```py
('stallCounter[start]', [-1.3699866305456414, 0.628122801542395])
('histogram[0][0]-binSize/2', -1.35)
('stallCounter[start] >= histogram[0][0]-binSize/2', True)
```
Essentially, Python 2 lets you make comparisons between a list and a number because it automatically gives you the 0-index number (or at least, I'm guessing that's what is happening). Python 3 is not happy with comparing a list and a number.

Reply:
It looks like a straight up bug in the Python 2 code.

Reply:
Yet another reason why this port is happening.

Reply:
I have no way of verifying if [0] is the right fix here; it certainly seems similar to the other lines. Since this was broken to begin with, I'm not sure if we have to block on finding the right thing, but in that case, a comment that explains the situation seems warranted

Reply:
After doing some testing, it seems as though python 2 always treats lists as always being "greater" than numbers.
Test:

```py
>>> a = []
>>> b = 2
>>> a > b
True
>>> a = [1]
>>> a > b
True
>>> b = 0
>>> a = []
>>> a > b
True
>>> a = [-1]
>>> a > b
True
```

Reply:
Okay, then one thing you could do is have your current line commented out, a comment saying what the deal is, and `if True:` with the rest. I don't know if the results of this script are deterministic, but if they are, it would allow us to run both Py2 and Py3 at the end of the month and see if there are any differences.

I don't know if that would be helpful or not. Just an idea.